src/lib.rs

```diff
@@ -167,12 +167,16 @@ mod builder;
 pub use builder::{KnownHosts, SessionBuilder};
 
 mod command;
-pub use command::{Command, OverSsh};
+pub use command::{Command as OwningCommand, OverSsh};
```

Comment:
I suggest renaming the type instead of creating a type alias, it would make our doc confusing to user

Reply:
Damn and here I'd thought I could avoid renaming `Command` everywhere.